Source/Blazorise/wwwroot/dropdown.js

```diff
@@ -0,0 +1,43 @@
+
+const _instances = [];
+
+export function initialize(element, elementId, options) {
+    element = getRequiredElement(element, elementId);
+
+    if (!element)
+        return;
+
+    const btnToggle = element.querySelector(".dropdown-toggle");
+    const menu = element.querySelector(".dropdown-menu");
+
+    //TODO : Set options / direction
+    const instance = Popper.createPopper(btnToggle, menu, {
```

Comment:
So, `Popper.createPopper` is called every time we need to show the menu? Can it be created once when the component is first created? Then let it handle itself. We do this pattern for all other components. Otherwise rename initialise/destroy to show/hide.

Reply:
Yea was kinda just trying it out. I already told you I commit stuff most of the times when I don't have more time haha Popper module